alf/environments/alf_wrappers.py

```diff
@@ -933,3 +933,34 @@ def _step(self, action):
         # action: [B, action_dim] or [action_dim]
         time_step = self._env.step(action)
         return time_step._replace(prev_action=prev_action)
+
+
```

Comment:
Would be good to have
`@alf.configurable` so gin config can also use this class.

Reply:
added